File: Arrays/LinearSearch.java
package Arrays;
import java.util.Scanner;
public class LinearSearch {
    public static void main(String[] args) {
        Scanner sc= new Scanner(System.in);
        int[] arr = new int[5];
        for(int i=0;i<5;i++)
            arr[i] = sc.nextInt();
        int key=sc.nextInt();
        for(int i=0;i<5;i++) {
            if (key == arr[i]) {
                System.out.println("Element found at position " + i);
                return;
            }
        }
        System.out.println("Element not found");
    }
}

File: Arrays/LinearSearch.cpp
#include <bits/stdc++.h>
using namespace std;
int LinearSearch(vector<int>&arr,int num){
    for(int i=0;i<arr.size();i++){
        if(arr[i]==num)
            return i;
    }
    return -1;
}
int main(){
    int n,a,key;
    cin>>n;
    cin>>key;
    vector<int>arr;
    for(int i=0;i<n;i++){
        cin>>a;
        arr.push_back(a);
    }
    cout<<"The key exists at position : "<<LinearSearch(arr,key);

}